dape.el

```diff
@@ -1055,18 +1059,19 @@ If `port' is not `:autoport' return config as is."
                  (unwind-protect
                      (process-contact port-probe :service)
                    (delete-process port-probe)))))
-  (when-let* ((command-args (plist-get config 'command-args))
-              (port (plist-get config 'port))
-              (port-string (number-to-string port)))
-    (plist-put
-     config
-     'command-args
-     (seq-map (lambda (arg)
-                (cond
-                 ((eq arg :autoport) port-string)
-                 ((stringp arg) (string-replace ":autoport" port-string arg))
-                 (t arg)))
-              command-args)))
+  (cl-loop for key in '(command-args command-env)
```

Comment:
nit: As it's not necessary to set `port` and `port-string` for each loop, I think the logic is better communicated with an surrounding `when-let*` expression, like:
```emacs-lisp
(when-let* ((port (plist-get config 'port))
            (port-string (number-to-string port)))
  (cl-loop for key in '(command-args command-env)
           for value = (plist-get config key)
           when value do
           ...
```

Reply:
Thanks for the feedback, this is great! I updated the PR accordingly. Let me know what you think!